packages/babel-preset-env/src/normalize-options.js

```diff
@@ -154,19 +146,7 @@ export const validateModulesOption = (
   return modulesOpt;
 };
 
-export const objectToBrowserslist = (object: Targets): Array<string> => {
```

Comment:
(NOTE for reviewers)

This has just been moved to `babel-preset-env/src/targets-parser.js`